src/realm-settings/KeysProvidersTab.tsx

```diff
@@ -285,7 +302,7 @@ export const KeysTabInner = ({ components, refresh }: KeysTabInnerProps) => {
           onDragStart={onDragStart}
           onDragMove={onDragMove}
           onDragCancel={onDragCancel}
-          itemOrder={itemOrder}
+          itemOrder={itemOrder as string[]}
```

Comment:
If the aforementioned changes are applied this can be removed.
```suggestion
          itemOrder={itemOrder}
```